src/lib/TranscriptEditor/Settings/index.js

```diff
@@ -66,6 +69,28 @@ class Settings extends React.Component {
             />
           </section>
 
+          <section className={ style.settingElement }>
+            <div className={ style.label }>Display Preview</div>
+            <Toggle
+              defaultValue={ this.props.previewIsDisplayed }
```

Comment:
PropTypes missing for a lot of things on this branch. About 30+ things to tidy up on `npm run lint` too

Reply:
ok fixed them all. 

```
/src/lib/TranscriptEditor/index.js
  284:18  warning  Arrow function should not return assignment  no-return-assign
  285:23  warning  Arrow function should not return assignment  no-return-assign
  286:23  warning  Arrow function should not return assignment  no-return-assign
```

these refer to this: 
```
<MediaPlayer
...
      hookSeek={ foo => this.setCurrentTime = foo }
      hookPlayMedia={ foo => this.playMedia = foo }
      hookIsPlaying={ foo => this.isPlaying = foo }
```

I have to admit I don't fully understand how this works, and what could be an alternative way to do it (from a syntax point of view) I think this patter was originally introduced by @Laurian (?)